features/steps/partcad-cli/commands/convert.py

```
@@ -0,0 +1,35 @@
+
+    src_path = os.path.join(base_src_path, src)
+    dest_path = os.path.join(context.test_dir, dest)
+
+    if not os.path.exists(src_path):
+        raise FileNotFoundError(f"Source file '{src_path}' does not exist. Check repository path: {base_src_path}")
+
+    shutil.copy(src_path, dest_path)
+    logging.debug(f"Copied {src_path} to {dest_path}")
+
+@then(u'a file named "{filename}" should exist')
+def step_impl(context, filename):
```

Ruff (0.8.2)

Redefinition of unused `step_impl` from line 14

(F811)